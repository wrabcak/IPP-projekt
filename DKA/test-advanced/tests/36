# epsilon uzaver
({s1, s2, s3, s4, s5, s6, s7, s8},
{'a', 'b', 'c'},
{
s1 'b' -> s2,

s2 'b' -> s6,

s3 'b' -> s4,
s3 'c' -> s1,

s4 'c' ->s5,
s4 'a' ->s2,

s5 'a' -> s5,

s7 'b' -> s6,

s8 'c' -> s7
},
s1
, {s6} )
