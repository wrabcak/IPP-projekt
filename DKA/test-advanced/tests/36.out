(
{qFALSE, s1, s2, s6},
{'a', 'b', 'c'},
{
qFALSE 'a' -> qFALSE,
qFALSE 'b' -> qFALSE,
qFALSE 'c' -> qFALSE,
s1 'a' -> qFALSE,
s1 'b' -> s2,
s1 'c' -> qFALSE,
s2 'a' -> qFALSE,
s2 'b' -> s6,
s2 'c' -> qFALSE,
s6 'a' -> qFALSE,
s6 'b' -> qFALSE,
s6 'c' -> qFALSE
},
s1,
{s6}
)